mmv1/products/certificatemanager/terraform.yaml

```diff
@@ -44,15 +44,34 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           cert_name: "dns-cert"
         ignore_read_extra:
           - "managed.0.dns_authorizations"
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "certificate_manager_self_managed_certificate"
+        primary_resource_id: "default"
+        vars:
+          cert_name: "self-managed-cert"
+        ignore_read_extra:
+          - "self_managed.0.pem_certificate"
+          - "self_managed.0.pem_private_key"
     properties:
       managed.dnsAuthorizations: !ruby/object:Overrides::Terraform::PropertyOverride
         # We don't support ignore_read on nested fields
         ignore_read: true
         custom_flatten: "templates/terraform/custom_flatten/certificate_manager_certificate_managed_dns_auth.go.erb"
-      selfManaged.certificatePem: !ruby/object:Overrides::Terraform::PropertyOverride
+      selfManaged: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Wouldn't adding ignore_read to the whole `self_managed` field work as well?
```
      selfManaged: !ruby/object:Overrides::Terraform::PropertyOverride
        sensitive: true
        ignore_read: true
```

If for some reason we need to add the custom_flatten function (e.g. `exactly_one_of` are always nested fields) then wouldn't it be enough to add it only to `self_managed` field? This way we would get rid of custom properties override for PEM fields.

Reply:
Yea! Ok, sorry, I've been hung up on something else and I've been wanting to take some time to think through this. You're right, I think we can actually remove everything here and just put the `ignore_read: true` on `selfManaged`. We don't even need the `custom_flatten` here or in the child attributes below anymore. I also think if we have `ignore_read: true` here, it automatically ignores the read on the test, so we can remove the whole thing from `ignore_read_extra` there.

So sorry for the confusion, things changed when the entire `selfManaged` block is returned with nothing in it, but my brain didn't take a step back there. Sorry for over-complicating this!

Reply:
done